jbpm-audit/src/main/java/org/jbpm/process/audit/query/AbstractAuditDeleteBuilderImpl.java

```diff
@@ -216,7 +223,7 @@ public T logBelongsToProcessInDeployment(String deploymentId) {
  
     abstract protected Class<?> getQueryType();
     
-    abstract protected String getQueryBase();
+    abstract protected String getQueryTable();
```

Comment:
this change is not necessary

Reply:
I can leave the method with current name but I need to change implementation. I need table name to build the proper jql statement, select or delete. Current method is allways returning a delete. I do not want to use string replacement. So I would say this change is very needed. Any alternative?